src/components/PageSettingsModal/ResourcePageSettingsModal.tsx

```diff
@@ -206,7 +212,9 @@ export const ResourcePageSettingsModal = ({
       <ModalContent>
         <ModalCloseButton id="settings-CLOSE" />
         <ModalHeader>
-          <Text as="h1">{!fileName ? "Create new page" : "Page settings"}</Text>
+          <Text as="h1">
+            {!fileName ? "Create new resource page" : "Resource page settings"}
```

Comment:
supernit - can we change this to
`{fileName ? "Resource page settings" : "Create new resource page"}` for clarity?

Reply:
Changed in https://github.com/isomerpages/isomercms-frontend/commit/36a32368a50a92cf4a0da92b1b2f795d49f0a0aa!